favorites.cs

```diff
@@ -0,0 +1,155 @@
+                
+                return new OkObjectResult(queryResultsFilter);
+
+            }
+            catch (Exception e)
+            {
+                log.LogError(e, "Problem loading");
+                return new BadRequestObjectResult("There was an issue");
+            }
+        }
+
+        [FunctionName("GetFavoritedItems")]
+        public async Task<IActionResult> RunGetFavoritedItems(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+                string itemId = req.Query["ItemId"];
+
+                Console.WriteLine($"ItemId {itemId}");
+
+                Pageable<TableEntity> queryResultsFilter = _favoriteTableClient.Query<TableEntity>(filter: $"ItemId eq {itemId}");
+
+                Console.WriteLine($"The query returned {queryResultsFilter.Count()} entities.");
+                return new OkObjectResult(queryResultsFilter);
+
+            }
+            catch (Exception e)
+            {
+                log.LogError(e, "Problem loading");
+                return new BadRequestObjectResult("There was an issue");
+            }
+        }
+
+        [FunctionName("MakeFavorite")]
+        public async Task<IActionResult> RunMakeFavorite(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+
+                string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+                log.LogInformation("Request body is {requestBody}", requestBody);
+                FavoriteItems favorites = JsonConvert.DeserializeObject<FavoriteItems>(requestBody);
+
+                await _favoriteTableClient.AddEntityAsync(favorites);
+
+                return new OkObjectResult(favorites);
+
+            }
+            catch (Exception e)
+            {
+                log.LogError(e, "Error with creating item. Check that it does not exist already.");
+            }
+
+            return new BadRequestObjectResult("There was an issue");
+        }
+        [FunctionName("RemoveFavorite")]
+        public async Task<IActionResult> RunRemoveFavorite(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "delete", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+                string PartitionKey = req.Query["PartitionKey"];
+                string RowKey = req.Query["RowKey"];
+                Console.WriteLine($"query params string is PartitionKey {PartitionKey} RowKey {RowKey}");
+
+                Pageable<TableEntity> favoriteFilter = _favoriteTableClient.Query<TableEntity>(filter: $"PartitionKey eq '{PartitionKey}' and RowKey eq '{RowKey}'");
+
+                Pageable<TableEntity> queryResultsFilter = _favoriteTableClient.Query<TableEntity>();
```

Comment:
If my understanding is correct, I believe these two line are not being used?

Reply:
removed :)